src/styles/sass/partials/_baseStyles.scss

```diff
@@ -0,0 +1,110 @@
+body {
+    background: 
+        url(../assets/pexels-kelly-l-2796105.jpg),
+        linear-gradient(rgba(0, 0, 0, 0.4),
+        rgba(0, 0, 0, 0.4));
+    background-blend-mode: saturation;
+    background-size: cover;
+    background-position: center;
+    height: 100vh;
+    color: white;
+    font-size: 125%;
+    text-align: center;
+    font-family: 'Montserrat', sans-serif;
+}
+
+.wrapper {
+    width: 90%;
+    max-width: 1080px;
+    margin: 0 auto;
+}
+
+h1 {
+    margin: 50px 0;
+    font-family: 'Limelight', cursive;
+    font-size: 3.5rem;
+}
+
+h2 {
+    font-family: 'Limelight', cursive;
+    font-size: 2.2rem;
+}
+
+p,
+label,
+li {
+    font-size: 1.2rem;
+}
+
+ul {
+    list-style: none;
+    padding: 0;
+}
+
+form,
+.drinkContainer {
+    background-color: rgba(0, 0, 0, 0.7);
+    border: 4px solid gold;
+    border-radius: 20px;
+    padding: 25px;
+}
+
+form {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-around;
+    margin: 50px auto;
+    height: 30vh;
+    select,
+    button {
+        font-family: 'Montserrat', sans-serif;
+        font-size: 1.4rem;
+        background-color: gold;
+        border: none;
+        border-radius: 15px;
+        margin-top: 15px;
+        padding: 10px;
+        text-align: center;
+        align-self: center;
+        cursor: pointer;
+    }
+    button {
+        font-weight: 600;
+        width: 70%;
+    }
+    select:hover,
+    select:focus,
+    button:hover,
+    button:focus-visible {
+        background-color: white;
+    }
+}
+
+.drinkContainer {
+    margin-bottom: 100px;
+}
+
+img {
+    display: block;
+    margin: 0 auto;
+    max-width: 100%;
+    max-height: 350px;
+}
+
+footer {
+    margin: 0;
+    background-color: black;
+    position: fixed;
+    bottom: 0;
+    width: 100%;
+    p {
+        font-size: 1rem;
```

Comment:
Switching between rem and pixels. You might consider a standard unit of measurement if possible